packages/api/src/axelar-config/isomorphic.ts

```diff
@@ -12,8 +12,14 @@ export class AxelarConfigClient extends RestService {
   }
 
   async getAxelarConfigs(env: Environment) {
+    // TODO: use 1.x for all envs once ayush fixes the config
```

Comment:
@SGiaccobasso keep an eye on this as well. we need to update once @blockchainguyy fixes the bug [here](https://interoplabs.slack.com/archives/C07URLT9N3V/p1741871108377019?thread_ts=1741850984.025499&cid=C07URLT9N3V)